.travis.yml

```diff
@@ -23,7 +24,7 @@ before_install:
   - cd WcaOnRails/
 before_script:
   - bundle exec rake db:reset RAILS_ENV=test
-sudo: false
+sudo: required
```

Comment:
It is strange. I thought MySQL would be extremely popular. But on the Trusty CI environment, if you look at data stores, MySQL is missing. https://docs.travis-ci.com/user/trusty-ci-environment

I'm going away for a week but when I get back I'll try to get the PR finished up so we can merge it up and then move on to fixing the registration flow.